pkg/osbuild/bootupd_stage_test.go

```diff
@@ -222,21 +222,21 @@ func TestGenBootupdDevicesMountsHappy(t *testing.T) {
 	})
 	assert.Equal(t, mounts, []osbuild.Mount{
 		{
-			Name:      "part4",
+			Name:      "-",
```

Comment:
Can this be named root? I see '-' there and my brain says 'stdout?'

Reply:
Yeah, I'm not a fan either, it's just us following the systemd mount convention:

```
$ sudo systemctl | grep '\.mount'
  -.mount                         loaded active mounted   Root Mount
  boot-efi.mount                  loaded active mounted   /boot/efi
  boot.mount                      loaded active mounted   /boot
  dev-hugepages.mount             loaded active mounted   Huge Pages File System
  dev-mqueue.mount                loaded active mounted   POSIX Message Queue File System
  home.mount                      loaded active mounted   /home
  proc-sys-fs-binfmt_misc.mount   loaded active mounted   Arbitrary Executable File Formats File System
  run-user-1000-doc.mount         loaded active mounted   /run/user/1000/doc
  run-user-1000-gvfs.mount        loaded active mounted   /run/user/1000/gvfs
  run-user-1000.mount             loaded active mounted   /run/user/1000
  sys-fs-fuse-connections.mount   loaded active mounted   FUSE Control File System
  sys-kernel-config.mount         loaded active mounted   Kernel Configuration File System
  sys-kernel-debug.mount          loaded active mounted   Kernel Debug File System
  sys-kernel-tracing.mount        loaded active mounted   Kernel Trace File System
  tmp-.mount_jetbraXWOO5X.mount   loaded active mounted   /tmp/.mount_jetbraXWOO5X
  tmp.mount                       loaded active mounted   Temporary Directory /tmp
  var-lib-nfs-rpc_pipefs.mount    loaded active mounted   RPC Pipe File System
```

Reply:
Oh geez.